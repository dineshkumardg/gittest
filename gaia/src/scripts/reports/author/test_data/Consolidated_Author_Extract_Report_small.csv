PSMID|Fulltitle|Article Sequence|Title|CHO Pub Type|Aucomposed|Prefix|First|Middle|Last|Suffix|Atlas Hub UID|PNA Notes
cho_book_1931_0000_001_0000|The International Gold Problem|1|Front Matter|||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|2|Front Matter|||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|3|Foreword||Neill Malcolm, Chairman of the Council of the Royal Institute of International Affairs||Neil||Malcolm|Chairman of the council of the Royal Institute of International Affairs|13735980|
cho_book_1931_0000_001_0000|The International Gold Problem|4|Introduction|||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|5|Front Matter|||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|6|Contents|||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|7|The International Functions of Gold||Sir Josiah Stamp, G. B. E.|Sir|Josiah||Stamp|G. B. E.|18027950|
cho_book_1931_0000_001_0000|The International Gold Problem|7||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|8|How Do we Want Gold to Behave?||Dennis H. Robertson||Dennis|H.|Robertson||13653683|
cho_book_1931_0000_001_0000|The International Gold Problem|8||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|9|Memorandum on Gold Production||Joseph Kitchin||Joseph||Kitchin||22207856|
cho_book_1931_0000_001_0000|The International Gold Problem|10|Gold Production||Joseph Kitchin||Joseph||Kitchin||22207856|
cho_book_1931_0000_001_0000|The International Gold Problem|10||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|11|How to Economize Gold||Sir Otto Niemeyer, G. B. E., K. C. B.|Sir|Otto||Niemeyer|G. B. E., K. C. B.|22212476|
cho_book_1931_0000_001_0000|The International Gold Problem|11||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|12|Some Problems of International Banking Policy||Dr. M. Palyi|Dr.|M.||Palyi||14854435|
cho_book_1931_0000_001_0000|The International Gold Problem|12||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|13|Memorandum on Legal Provisions Governing Reserves of Central Banks||C. H. Kisch, C. B.||C.|H.|Kisch|C. B.|19094355|
cho_book_1931_0000_001_0000|The International Gold Problem|14|Central Bank Reserves||C. H. Kisch, C. B.||C.|H.|Kisch|C. B.|19094355|
cho_book_1931_0000_001_0000|The International Gold Problem|14||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|15|For what Objects Do Central Banks Hold Gold?||The Hon. R. H. Brand, C. M. G.|The Hon.|R.|H.|Brand|C. M. G.||Can't identify
cho_book_1931_0000_001_0000|The International Gold Problem|15||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|16|The International Consequences of the Present Distribution of Gold Holdings||M. Charles Rist||M.|Charles|Rist||22207893|
cho_book_1931_0000_001_0000|The International Gold Problem|16||||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|17|Bibliography||Mrs. Annette Henderson B. Sc. (Econ.)|Mrs.|Annette||Henderson|B. Sc. (Econ.)||Can't identify.  P OCC could be Economist given the degree mentioned, but couldn't make that assumption.  
cho_book_1931_0000_001_0000|The International Gold Problem|18|Index|||||||||
cho_book_1931_0000_001_0000|The International Gold Problem|19|Back Matter|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|1|Front Matter|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|2|Front Matter|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|3|Foreword||Astor, Chairman of the Council||||Astor|Chairman of the Council||Can't identify
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|4|Introduction|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|5|Contents|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|6|Part I Foreign Policy|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|7|Section I European Interests|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|8|Chapter I Great Britain in Europe|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|9|Chapter II Great Britain and Western Europe to 1935|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|10|Chapter III Great Britain and Western Europe since 1935|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|11|Chapter IV Other European Interests|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|12|Section II The Mediterranean and the Middle East|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|13|Chapter V Mediterranean Interests|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|14|Chapter VI Middle Eastern Interests : I|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|15|Chapter VII Middle Eastern Interests : II|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|16|Section III The Far East ; The American Continent|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|17|Chapter VIII The Far East|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|18|Chapter IX The American Continent|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|19|Part II Imperial Defence|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|20|Chapter X The Problem|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|21|Chapter XI Military Forces and Expenditure|||||||||
cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|22|Index|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|1|Front Matter|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|2|Front Matter|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|3|Contents|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|4|Preface||Major General Sir Ian Jacob K. B. E., C. B.|Major General Sir|Ian||Jacob|K. B. E., C. B.|22208019|
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|5|I The Problem of the Democracies|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|6|II &apos;All Mischief Short of War&apos;|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|7|III Policies of Defence|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|8|IV The Atomic Bomb|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|9|V Western Europe-The Citadel|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|10|VI The Defence Pacts-Theory and Fact|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|11|VII The Forces Required|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|12|VIII The Role of the Germans|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|13|IX Can Rearmament Be Afforded?|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|14|X The Struggle of Ideas|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|15|XI Years of Effort|||||||||
cho_book_1950_mclachlan_000_0000|Defence in the Cold War|16|Some Chatham House Publications|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|Book Level|||Carol Ann Cosgrove||Carol|Ann|Cosgrove||14341480|
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|1|Front Matter|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|2|FrontMatter|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|3|Contents|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|4|FrontMatter|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|5|Introduction|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|6|I.BackgroundBooksonWesternEuropeanIntegration|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|7|II.FirstStepsinWesternEuropeanIntegration|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|8|III.TheEuropeanCommunities|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|9|IV.TheFirstBritishApplication1961-3|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|10|V.TheCurrentApplication|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|11|VI.PossibleAlternativesforBritain|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|12|Index|||||||||
cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|13|BackMatter|||||||||
