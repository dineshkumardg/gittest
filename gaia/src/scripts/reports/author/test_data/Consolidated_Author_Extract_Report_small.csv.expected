Product Content Type|Asset ID|PSMID|Fulltitle|Article Sequence|Title|CHO Pub Type|Aucomposed|Prefix|First|Middle|Last|Suffix|Atlas Hub UID|PNA Notes
Books|SXBWCA324506771|cho_book_1931_0000_001_0000|The International Gold Problem|1|Front Matter|||||||||
Books|MPXEIO935060819|cho_book_1931_0000_001_0000|The International Gold Problem|2|Front Matter|||||||||
Books|PJIBLV871358750|cho_book_1931_0000_001_0000|The International Gold Problem|3|Foreword||Neill Malcolm, Chairman of the Council of the Royal Institute of International Affairs||Neil||Malcolm|Chairman of the council of the Royal Institute of International Affairs|13735980|
Books|VGXDRI405381350|cho_book_1931_0000_001_0000|The International Gold Problem|4|Introduction|||||||||
Books|QFLMSJ382929248|cho_book_1931_0000_001_0000|The International Gold Problem|5|Front Matter|||||||||
Books|YSLAVL712663596|cho_book_1931_0000_001_0000|The International Gold Problem|6|Contents|||||||||
Books|HPXGKR535776524|cho_book_1931_0000_001_0000|The International Gold Problem|7|The International Functions of Gold||Sir Josiah Stamp, G. B. E.|Sir|Josiah||Stamp|G. B. E.|18027950|
Books|HPXGKR535776524|cho_book_1931_0000_001_0000|The International Gold Problem|7||||||||||
Books|PRDUOG578237927|cho_book_1931_0000_001_0000|The International Gold Problem|8|How Do we Want Gold to Behave?||Dennis H. Robertson||Dennis|H.|Robertson||13653683|
Books|PRDUOG578237927|cho_book_1931_0000_001_0000|The International Gold Problem|8||||||||||
Books|EIURME812589680|cho_book_1931_0000_001_0000|The International Gold Problem|9|Memorandum on Gold Production||Joseph Kitchin||Joseph||Kitchin||22207856|
Books|YCTBTB736723261|cho_book_1931_0000_001_0000|The International Gold Problem|10|Gold Production||Joseph Kitchin||Joseph||Kitchin||22207856|
Books|YCTBTB736723261|cho_book_1931_0000_001_0000|The International Gold Problem|10||||||||||
Books|WYHORO177347095|cho_book_1931_0000_001_0000|The International Gold Problem|11|How to Economize Gold||Sir Otto Niemeyer, G. B. E., K. C. B.|Sir|Otto||Niemeyer|G. B. E., K. C. B.|22212476|
Books|WYHORO177347095|cho_book_1931_0000_001_0000|The International Gold Problem|11||||||||||
Books|ENDLER677650555|cho_book_1931_0000_001_0000|The International Gold Problem|12|Some Problems of International Banking Policy||Dr. M. Palyi|Dr.|M.||Palyi||14854435|
Books|ENDLER677650555|cho_book_1931_0000_001_0000|The International Gold Problem|12||||||||||
Books|XCZDDS719012329|cho_book_1931_0000_001_0000|The International Gold Problem|13|Memorandum on Legal Provisions Governing Reserves of Central Banks||C. H. Kisch, C. B.||C.|H.|Kisch|C. B.|19094355|
Books|JKEOMI241082687|cho_book_1931_0000_001_0000|The International Gold Problem|14|Central Bank Reserves||C. H. Kisch, C. B.||C.|H.|Kisch|C. B.|19094355|
Books|JKEOMI241082687|cho_book_1931_0000_001_0000|The International Gold Problem|14||||||||||
Books|MWEBTT341318081|cho_book_1931_0000_001_0000|The International Gold Problem|15|For what Objects Do Central Banks Hold Gold?||The Hon. R. H. Brand, C. M. G.|The Hon.|R.|H.|Brand|C. M. G.||Can't identify
Books|MWEBTT341318081|cho_book_1931_0000_001_0000|The International Gold Problem|15||||||||||
Books|SDIEVG943330743|cho_book_1931_0000_001_0000|The International Gold Problem|16|The International Consequences of the Present Distribution of Gold Holdings||M. Charles Rist||M.|Charles|Rist||22207893|
Books|SDIEVG943330743|cho_book_1931_0000_001_0000|The International Gold Problem|16||||||||||
Books|TIRCZB777322352|cho_book_1931_0000_001_0000|The International Gold Problem|17|Bibliography||Mrs. Annette Henderson B. Sc. (Econ.)|Mrs.|Annette||Henderson|B. Sc. (Econ.)||Can't identify.  P OCC could be Economist given the degree mentioned, but couldn't make that assumption.  
Books|JJKLSI844339819|cho_book_1931_0000_001_0000|The International Gold Problem|18|Index|||||||||
Books|STIQOK154779078|cho_book_1931_0000_001_0000|The International Gold Problem|19|Back Matter|||||||||
Books|LRTBRJ999852333|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|1|Front Matter|||||||||
Books|XYHKYZ145618031|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|2|Front Matter|||||||||
Books|FYLQKR304981122|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|3|Foreword||Astor, Chairman of the Council||||Astor|Chairman of the Council||Can't identify
Books|VHNSAV102414365|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|4|Introduction|||||||||
Books|VSVKNV541634623|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|5|Contents|||||||||
Books|SVMBQA690611846|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|6|Part I Foreign Policy|||||||||
Books|AGEBWE313455247|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|7|Section I European Interests|||||||||
Books|EWUTET197090103|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|8|Chapter I Great Britain in Europe|||||||||
Books|NDBGYP948852187|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|9|Chapter II Great Britain and Western Europe to 1935|||||||||
Books|BJDHZS530488584|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|10|Chapter III Great Britain and Western Europe since 1935|||||||||
Books|VEPCRH614051849|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|11|Chapter IV Other European Interests|||||||||
Books|VFSDKN960983082|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|12|Section II The Mediterranean and the Middle East|||||||||
Books|TBVVJY225968528|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|13|Chapter V Mediterranean Interests|||||||||
Books|HCPNVI324416661|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|14|Chapter VI Middle Eastern Interests : I|||||||||
Books|LLHUMG025787492|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|15|Chapter VII Middle Eastern Interests : II|||||||||
Books|SQBGMV467070527|cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|16|Section III The Far East ; The American Continent|||||||||
Books||cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|17|Chapter VIII The Far East|||||||||
Books||cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|18|Chapter IX The American Continent|||||||||
Books||cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|19|Part II Imperial Defence|||||||||
Books||cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|20|Chapter X The Problem|||||||||
Books||cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|21|Chapter XI Military Forces and Expenditure|||||||||
Books||cho_book_1940_0000_001_0000|Political and Strategic Interests of the United Kingdom an Outline|22|Index|||||||||
Books|ERNTMW370255362|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|1|Front Matter|||||||||
Books|HJJOPE412732288|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|2|Front Matter|||||||||
Books|CMOMHI029372086|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|3|Contents|||||||||
Books|VSQOAX195190407|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|4|Preface||Major General Sir Ian Jacob K. B. E., C. B.|Major General Sir|Ian||Jacob|K. B. E., C. B.|22208019|
Books|UAYPFA813295704|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|5|I The Problem of the Democracies|||||||||
Books|JGPLIQ630584586|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|6|II &apos;All Mischief Short of War&apos;|||||||||
Books|YEGYAR813514451|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|7|III Policies of Defence|||||||||
Books|MWDYJA333319999|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|8|IV The Atomic Bomb|||||||||
Books|CULVBW583302787|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|9|V Western Europe-The Citadel|||||||||
Books|CTSWWM999172542|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|10|VI The Defence Pacts-Theory and Fact|||||||||
Books|CSMXKW737841842|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|11|VII The Forces Required|||||||||
Books|VABMWR395396369|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|12|VIII The Role of the Germans|||||||||
Books|USNINN647045556|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|13|IX Can Rearmament Be Afforded?|||||||||
Books|XXNPBB984773523|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|14|X The Struggle of Ideas|||||||||
Books|DCLKGM935087511|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|15|XI Years of Effort|||||||||
Books|YPQPPX363818004|cho_book_1950_mclachlan_000_0000|Defence in the Cold War|16|Some Chatham House Publications|||||||||
Books||cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|Book Level|||Carol Ann Cosgrove||Carol|Ann|Cosgrove||14341480|
Books|HNRZBF767642820|cho_book_1970_cosgrove_000_0000||1
Books|WMVPRV989596489|cho_book_1970_cosgrove_000_0000||2
Books|SGKSDP699960323|cho_book_1970_cosgrove_000_0000||3
Books|DSUQJC584877520|cho_book_1970_cosgrove_000_0000||4
Books|PARSAH782049285|cho_book_1970_cosgrove_000_0000||5
Books|WHVPWH635125629|cho_book_1970_cosgrove_000_0000||6
Books|NZMDDK986340136|cho_book_1970_cosgrove_000_0000||7
Books|DEGOKK042695545|cho_book_1970_cosgrove_000_0000||8
Books|IGVPGT544765704|cho_book_1970_cosgrove_000_0000||9
Books|XRSIWT718797953|cho_book_1970_cosgrove_000_0000||10
Books|TCKTWE120216395|cho_book_1970_cosgrove_000_0000||11
Books|NHRKTV111230022|cho_book_1970_cosgrove_000_0000||12
Books|ZPXRNH457171781|cho_book_1970_cosgrove_000_0000||13
Books|WGJURZ865936144|cho_book_1970_cosgrove_000_0000||14
Books|HNRZBF767642820|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|1|Front Matter|||||||||
Books|WMVPRV989596489|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|2|FrontMatter|||||||||
Books|SGKSDP699960323|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|3|Contents|||||||||
Books|DSUQJC584877520|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|4|FrontMatter|||||||||
Books|PARSAH782049285|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|5|Introduction|||||||||
Books|WHVPWH635125629|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|6|I.BackgroundBooksonWesternEuropeanIntegration|||||||||
Books|NZMDDK986340136|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|7|II.FirstStepsinWesternEuropeanIntegration|||||||||
Books|DEGOKK042695545|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|8|III.TheEuropeanCommunities|||||||||
Books|IGVPGT544765704|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|9|IV.TheFirstBritishApplication1961-3|||||||||
Books|XRSIWT718797953|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|10|V.TheCurrentApplication|||||||||
Books|TCKTWE120216395|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|11|VI.PossibleAlternativesforBritain|||||||||
Books|NHRKTV111230022|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|12|Index|||||||||
Books|ZPXRNH457171781|cho_book_1970_cosgrove_000_0000|A Reader&apos;s Guide to Britain and the European Communities|13|BackMatter|||||||||
